src/components/input/input.e2e.ts

```diff
@@ -1350,25 +1359,139 @@ describe("calcite-input", () => {
     expect(await element.getProperty("value")).toBe("-123");
   });
 
-  describe("is form-associated", () => {
-    it("supports type=text", () => formAssociated("calcite-input", { testValue: "test" }));
-    it("supports type=number", () => formAssociated("<calcite-input type='number'></calcite-input>", { testValue: 5 }));
-  });
+  describe("ArrowUp/ArrowDown function as Home/End within calcite-input", () => {
```

Comment:
To avoid association with `Home`/`End` keys (e.g., scrolling page), can you replace `Home/End` with something that describes the caret behavior?